editor/gui/editor_quick_open_dialog.h

```diff
@@ -266,5 +273,11 @@ class EditorQuickOpenDialog : public AcceptDialog {
 
 	Callable item_selected_callback;
 
+	Object *property_object = nullptr;
+	String property_path;
+	Variant initial_property_value;
```

Comment:
```suggestion
	Object *property_object = nullptr;
	String property_path;
	Variant initial_property_value;

```

Reply:
Sorry for a perhaps-silly question, but the suggestion view from GitHub here is a bit confusing for me - is this suggestion just changing the spacing between groups of class members, or something else? For the most part the code looks identical, but I see there's some whitespace highlighted as being different as well...